lib/erlef_web/templates/blog/index.html.eex

```diff
@@ -1,56 +1,60 @@
-<div class="container mt-2">
-  <% post1 = Enum.at(@posts, 0) %>
-  <%= unless is_nil(post1) do %>
-    <div class="jumbotron p-4 p-md-5 text-white rounded bg-dark">
-      <div class="col-md-6 px-0">
-        <h1 class="display-4 font-italic"><%= post1.title %></h1>
-        <p class="lead my-3"><%= post1.excerpt %></p>
-        <p class="lead mb-0">
-        <%= link("Continuing Reading", to: Routes.news_path(@conn, :show, @topic, post1.slug), class: "stretched") %>
+<%= if length(@posts) > 0 do %>
```

Comment:
The problem with using `length` here is that the list could be really big. When we call `length` on a list it must traverse the list to accumulate the length. A better solution would probably be to say `if @posts == [] do`. What do you think?

Reply:
Oh, pretty neat trick. Cool. Agree.